# Message
float64 secs
float64[] mass_matrix
float64[] damping_matrix
float64[] thrusters_dynamics
float64[] tau
float64[] coriolis_matrix
float64[] c_v
float64[] gravity
float64[] v_dot
float64[] v
float64[] p_dot
float64[] p